Index: src/test/java/utils/RandomizerData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/utils/RandomizerData.java b/src/test/java/utils/RandomizerData.java
new file mode 100644
--- /dev/null	(date 1746424776355)
+++ b/src/test/java/utils/RandomizerData.java	(date 1746424776355)
@@ -0,0 +1,49 @@
+//package utils;
+//import com.github.javafaker.Faker;
+//
+//import java.util.Locale;
+//
+//public class RandomizerData {
+////import java.util.Locale;
+//
+//
+//    Faker faker = new Faker(new Locale("en"));
+//
+//
+//    public String firstName = faker.name().firstName(),
+//            lastName = faker.name().lastName(),
+//            userEmail = faker.internet().emailAddress(),
+//            gender = faker.options().option("Male", "Female", "Other"),
+//            phoneNumber = faker.phoneNumber().subscriberNumber(10),
+//            dayOfBirth = String.format("%s", faker.number().numberBetween(1, 28)),
+//            monthOfBirth = faker.options().option("January", "February", "March", "April",
+//                    "May", "June", "July", "August", "September", "October", "November", "December"),
+//            yearOfBirth = String.format("%s", faker.number().numberBetween(1924, 2010)),
+//            subject = faker.options().option("Hindi", "English", "History", "Maths", "Physics", "Chemistry", "Biology"),
+//            hobby = faker.options().option("Sports", "Reading", "Music"),
+//            picture = faker.options().option("Test.jpg"),
+//            address = faker.address().fullAddress(),
+//            state = faker.options().option("NCR", "Uttar Pradesh", "Haryana", "Rajasthan"),
+//            city = selectCity(state);
+//
+//
+//    public String selectCity(String state) {
+//        switch (state) {
+//
+//            case "NCR":
+//                return faker.options().option("Delhi", "Gurgaon", "Noida");
+//
+//            case "Uttar Pradesh":
+//                return faker.options().option("Agra", "Lucknow", "Merrut");
+//
+//            case "Haryana":
+//                return faker.options().option("Karnal", "Panipat");
+//
+//            case "Rajasthan":
+//                return faker.options().option("Jaipur", "Jaiselmer");
+//
+//            default:
+//                return null;
+//        }
+//    }
+//}
